frontend/src/components/pages/touristicContent/useTouristicContent.tsx

```diff
@@ -20,11 +22,25 @@ export const useTouristicContent = (
   const id = isTouristicContentUrlString ? touristicContentUrl.split('-')[0] : '';
   const path = isTouristicContentUrlString ? decodeURI(touristicContentUrl) : '';
   const router = useRouter();
+
+  const { data: commonDictionaries } = useQuery<CommonDictionaries, Error>(
+    ['commonDictionaries', language],
+    () => getCommonDictionaries(language),
+    {
+      onError: async error => {
+        if (isRessourceMissing(error)) {
+          await router.push(routes.HOME);
+        }
+      },
+      staleTime: ONE_DAY / 2,
+    },
+  );
+
```

Comment:
Lots of duplicate code, can't this logic be abstracted in a file?

Reply:
Good point, i DRY this part with a dedicated file :+1: